feature/authentication/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+<!--    Login-->
+    <dimen name="textSize_login_welcomeTitle">24dp</dimen>
+    <dimen name="textSize_login_forgotPasswordTextButton">14dp</dimen>
+    <dimen name="textSize_login_loginButton">16dp</dimen>
+    <dimen name="textSize_login_registerTextButton">16dp</dimen>
+    <dimen name="textSize_login_loginTextField">13dp</dimen>
+    <dimen name="textSize_login_loginErrorText">9dp</dimen>
```

Comment:
hmm đoạn này hơi lạ. textSize thường là sp. Có gì bắt buộc e phải dùng dp ko?

Reply:
Vì compose không get thẳng giá trị dimen ra set text size được nên em phải tạo 1 hàm để get text size, mà hàm đó nhận giá trị dp rồi convert thành sp, nên em buộc phải dùng dp, nếu dùng sp nó sẽ convert sai và chữ trở nên to đùng

Reply:
thay đổi cỡ chữ nghĩa là sao anh nhỉ, tức là app có chức năng thay đổi làm cỡ chữ lớn hơn ấy ạ, vậy thì em nghĩ xử lý giống với xml thường được, sau này nếu có vụ đó thì em nghiên cứu thêm sau, tạm thời em resolve cmt này nhé

Reply:
nó ko phải chức năng của app, mà chức năng của hệ thống. Khi e để font chữ là sp thì e chỉnh cỡ chữ trong settings của đt, thì font chữ trong app của mình cũng thay đổi luôn. E resolve luôn cũng đc, chắc app mình về sau publish thì may ra mới tính đến phần này

Reply:
nhớ mình có best practice là chữ phải để sp để khi user họ thay đổi trong setting của máy thì chữ to lên là đc

Reply:
nếu e để ý có 1 số người già họ chỉnh font chữ trên máy của họ to lên ấy. Nếu mình để dp thì chữ trong app mình vẫn bé như vậy, ko chỉnh to lên khi đổi settings của máy 

![image](https://user-images.githubusercontent.com/33829681/225206099-4f969266-5d09-4f57-a55b-68923794cd15.png)

Reply:
https://stackoverflow.com/a/72306940/10325347